geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt

```diff
@@ -10,6 +10,13 @@ Version 2.8.1
    Use QGIS server 3.28 with GDAL 3.7 (instead of 3.6)
    Use GDAL 3.7 as based image (instead of 3.6)
 
+2. Docker Compose version 2
+
+   Support of Docker Compose version 2 (with the command `docker compose` instead of `docker-compose`) is now supported.
```

Comment:
either
Docker Compose version 2 (...) is now supported.
or
Support of Docker Compose version 2 (...) has been implemented.